server/mscalendar/availability.go

```diff
@@ -212,7 +212,15 @@ func (m *mscalendar) deliverReminders(users []*store.User, calendarViews []*remo
 		}
 
 		mattermostUserID := usersByRemoteID[view.RemoteUserID].MattermostUserID
-		m.notifyUpcomingEvents(mattermostUserID, view.Events)
+		if fetchIndividually {
+			engine, err := m.FilterCopy(withActingUser(user.MattermostUserID))
+			if err != nil {
+				m.Logger.With(bot.LogContext{"err": err}).Errorf("error getting engine for user")
+			}
+			engine.notifyUpcomingEvents(mattermostUserID, view.Events)
```

Comment:
Should we return when this error happens?

Reply:
Yeah, I put a `continue` in there to avoid trying to notify that user. Good catch.